docs/develop/java/spring/redis-om/index-redis-om-spring.mdx

```diff
@@ -0,0 +1,53 @@
+---
+id: redis-om-spring
+title: Redis OM - Spring
+sidebar_label: Introduction
+slug: /develop/java/spring/redis-om/redis-om-spring
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import RedisCard from '@site/src/theme/RedisCard';
+
+
+## Introduction
+
+The Redis OM family of projects aim is to provide high-level abstractions idiomatically implemented for
+ your language/platform of choice. We currently cater to the Node, Python, .Net and Spring communities.
+
+The Spring Framework is the leading full-stack Java/JEE application framework and Redis OM Spring (ROMS)
+goal is to enable developers to easily add the power of Redis to their Spring Boot applications.
+
+Redis OM Spring provides powerful repository and custom object-mapping abstractions built on top of the
+powerful Spring Data Redis (SDR) framework.
+
+The current **preview** release provides all of SDRs capabilities plus:
+
+* A `@Document` annotation to map Spring Data models to Redis JSON documents
+* Enhances SDR's `@RedisHash` via `@EnableRedisEnhancedRepositories` to:
+  - use Redis' native search engine (RediSearch) for secondary indexing
+  - use [ULID](https://github.com/ulid/spec) indentifiers for `@Id` annotated fields
+* `RedisDocumentRepository` with automatic implementation of Repository interfaces for complex querying capabilities using `@EnableRedisDocumentRepositories`
+* Declarative Search Indices via `@Indexable`
+* Full-text Search Indices via `@Searchable`
+* `@Bloom` annotation to determine very fast, with and with high degree of certainty, whether a value is in a collection
```

Comment:
determine very quickly... or fastly... or rapidly. Adverb instead of an adjective.